packages/@sanity/desk-tool/src/components/confirmDeleteDialog/ConfirmDeleteDialogBody.tsx

```diff
@@ -33,6 +39,50 @@ function getSchemaType(typeName: string): SchemaType | null {
   return schema.get(typeName) || null
 }
 
+const EMPTY_ARRAY: [] = []
```

Comment:
Can I suggest moving the `ReferenceLink` function to a separate file? This file is becoming quite large

Reply:
Good point, I moved it to a separate file 👍